jwala-persistence/src/main/java/com/cerner/jwala/persistence/service/impl/JpaJvmPersistenceServiceImpl.java

```diff
@@ -257,16 +255,8 @@ public int removeTemplate(final String jvmName, final String templateName) {
     public boolean checkJvmResourceFileName(final String groupName, final String jvmName, final String fileName) {
         return jvmCrudService.checkJvmResourceFileName(groupName, jvmName, fileName);
     }
+<<<<<<< HEAD
```

Comment:
Hi @rahulsayini ,

This will break the build. I think you were not able to resolve the conflict cleanly. Can we pair together and work on your branch to see what is happening ?